packages-exp/auth-exp/src/core/strategies/custom_token.test.ts

```diff
@@ -21,14 +21,22 @@ import * as chaiAsPromised from 'chai-as-promised';
 import { OperationType } from '../../model/enums';
 
 import { mockEndpoint } from '../../../test/helpers/api/helper';
-import { testAuth, TestAuth } from '../../../test/helpers/mock_auth';
+import { testAuth, TestAuth, testUser } from '../../../test/helpers/mock_auth';
 import * as mockFetch from '../../../test/helpers/mock_fetch';
 import { Endpoint } from '../../api';
 import { APIUserInfo } from '../../api/account_management/account';
 import { IdTokenResponse, IdTokenResponseKind } from '../../model/id_token';
 import { UserCredentialInternal } from '../../model/user';
-import { signInWithCustomToken } from './custom_token';
+import {
+  clearCustomTokenProvider,
+  setCustomTokenProvider,
+  signInWithCustomToken
+} from './custom_token';
+import { spy } from 'sinon';
```

Comment:
small nit-- for consistency with other tests, please `import * as sinon from 'sinon'` and use `sinon.spy()` in tests